extensions/sql-migration/src/api/azure.ts

```diff
@@ -166,12 +166,24 @@ export async function startDatabaseMigration(account: azdata.Account, subscripti
 		throw new Error(response.errors.toString());
 	}
 	return {
-		errors: response.errors,
 		status: response.response.status,
 		databaseMigration: response.response.data
 	};
 }
 
+export async function getMigrationStatus(account: azdata.Account, subscription: Subscription, migration: DatabaseMigration): Promise<any> {
+	const api = await getAzureCoreAPI();
+	const host = `https://eastus2euap.management.azure.com`;
+	const path = `${migration.id}?$expand=MigrationStatusDetails&api-version=2020-09-01-preview`;
+	const response = await api.makeAzureRestRequest(account, subscription, path, azurecore.HttpRequestMethod.GET, undefined, true, host);
+	if (response.errors.length > 0) {
+		throw new Error(response.errors.toString());
```

Comment:
what does errors.toString() look like?

Reply:
It just converts the Json errors to a string.